Branch-TestBed/Branch-SDK-Tests/BNCServerInterfaceTests.m

```diff
@@ -19,176 +23,327 @@ @interface BNCServerInterfaceTests : XCTestCase
 
 @implementation BNCServerInterfaceTests
 
+#pragma mark - Tear Down
+- (void)tearDown
+{
```

Comment:
We've been following the syntax where the { is on the previous line with the method name. I'd prefer if we kept it for consistency. It also seems like this function is using a 2 space indentation, whereas the rest of the SDK uses 4.